Documentation/Books/Manual/ReleaseNotes/UpgradingChanges35.md

```diff
@@ -6,3 +6,6 @@ upgrading to ArangoDB 3.5, and adjust any client programs if necessary.
 
 The following incompatible changes have been made in ArangoDB 3.5:
 
+* Numerical primary index keys will be sorted in lexicographical order,
```

Comment:
But `_key` is never numeric? It can be a string using digit characters only, but it's still a string.